sycl/include/CL/sycl/detail/defines_elementary.hpp

```diff
@@ -90,7 +92,7 @@
 // define __SYCL_WARNING convenience macro to report compiler warnings
 #if defined(__GNUC__)
 #define __SYCL_GCC_PRAGMA(x) _Pragma(#x)
-#define __SYCL_WARNING(msg) __SYCL_GCC_PRAGMA(GCC warning msg)
+#define __SYCL_WARNING(msg) __SYCL_GCC_PRAGMA(message msg)
```

Comment:
Modified it to the message to align with definitions of this macro for other environments(clang, msvc). The name of the macro seems to be incorrect though.